integration-tests/features/steps/common.py

```diff
@@ -982,7 +982,8 @@ def stack_analysis_check_replaces(json_data, component, version, replaced_by, re
 
 
 @then('I should find that the component {component} version {version} has only one replacement')
-def stack_analysis_check_replaces_count(json_data, component, version):
+@then('I should find that the component {component} version {version} has {expected_replacements:d} replacements')
```

Comment:
@tisnik let's not put any count here because we can't really predict the `expected_replacements` from the analytics. The only thing that we can test is whether or not the components in the `replaces` block are part of the user's original components list.